modules/serverless-rn-1-33-0.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-33-0_{context}"]
+= Red Hat {ServerlessProductName} 1.33
+
+{ServerlessProductName} 1.33 is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes.
+
+[id="new-features-1-33-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.12.
+* {ServerlessProductName} now uses Knative Eventing 1.12.
+* {ServerlessProductName} now uses Kourier 1.12.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.12.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.12.
+* The `kn func` CLI plugin now uses `func` 1.13.
+
```

Comment:
Blank line:
```suggestion

```
And a few others below for this list.

Reply:
We visually separate this "template" part with just new component versions from "new features proper" part in all release notes for Serverless, so this is OK. Same for newline buffering of each non-just-component-version release note. I've suggested to remove the double empty line below.

Reply:
@briandooley Don't accept this particular suggestion.